web-portal/pages/submit-event.vue

```diff
@@ -45,10 +45,27 @@
             <h1 class="centered-header">
               Hmmm... something went wrong :(
             </h1>
-            <p style="text-align: center">
-              Please ping the management at
-              <a href="mailto:info@infinite.industries">info@infinite.industries</a>.
-            </p>
+
+            <div class="infinite-submission-form__error-info-section">
+              <p
+                class="infinite-submission-form__error-message"
+                v-if="submissionErrorMessage !== null"
+              >
+                {{submissionErrorMessage}}
+              </p>
+
+              <p class="infinite-submission-form__report-problem-message">
+                Please ping the management at or go back and try again
+                <a href="mailto:info@infinite.industries">info@infinite.industries</a>.
```

Comment:
The email doesn't fit into the sentence anymore; assume what we want here is more like
```suggestion
                Please ping the management at <a href="mailto:info@infinite.industries">info@infinite.industries</a> or go back and try again.
```

Reply:
fixed c1cccb8e64d0d5d9cadf403f455142fc40c70283